demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -1013,51 +1021,36 @@ def is_context_change_in_readme(self) -> bool:
         """
         valid = True
-        # the pattern to get the context part out of command section:
-        context_section_pattern = CONTEXT_OUTPUT_README_TABLE_HEADER.replace('|', '\\|').replace('*', r'\*') + ".(.*?)#{3,5}"
-        # the pattern to get the value in the first column under the outputs table:
-        context_path_pattern = r"\| ([^\|]*) \| [^\|]* \| [^\|]* \|"
 
         dir_path = os.path.dirname(self.file_path)
         if not os.path.exists(os.path.join(dir_path, 'README.md')):
             return True
 
-        # get README file's content
         readme_path = os.path.join(dir_path, 'README.md')
+        if f'{readme_path} - [RM102]' in FOUND_FILES_AND_IGNORED_ERRORS \
+                or f'{readme_path} - [RM102]' in FOUND_FILES_AND_ERRORS \
+                or f'{self.file_path} - [IN136]' in FOUND_FILES_AND_IGNORED_ERRORS \
+                or f'{self.file_path} - [IN136]' in FOUND_FILES_AND_ERRORS:
+            return False
+
+        # get README file's content
         with open(readme_path, 'r') as readme:
             readme_content = readme.read()
-            readme_content += "### "  # mark end of file so last pattern of regex will be recognized.
-
-        commands = self.current_file.get("script", {}).get('commands', [])
-
-        for command in commands:
-            command_name = command.get('name')
-
-            # Gets all context path in the relevant command section from README file
-            command_section_pattern = fr" Base Command..`{command_name}`.(.*?)\n### "  # pattern to get command section
-            command_section = re.findall(command_section_pattern, readme_content, re.DOTALL)
-            if not command_section:
-                continue
-            if not command_section[0].endswith('###'):
-                command_section[0] += '###'  # mark end of file so last pattern of regex will be recognized.
-            context_section = re.findall(context_section_pattern, command_section[0], re.DOTALL)
-            if not context_section:
-                context_path_in_command = set()
-            else:
-                context_path_in_command = set(re.findall(context_path_pattern, context_section[0], re.DOTALL))
-                context_path_in_command.remove('---')
 
-            existing_context_in_yml = set(extract_multiple_keys_from_dict("contextPath", command))
-
-            # finds diff between YML and README
-            only_in_yml_paths = existing_context_in_yml - context_path_in_command
-            only_in_readme_paths = context_path_in_command - existing_context_in_yml
-            if only_in_yml_paths:
-                error, code = Errors.readme_missing_output_context(command_name, ", ".join(only_in_yml_paths))
+        # commands = self.current_file.get("script", {}).get('commands', [])
+        difference = compare_context_path_in_yml_and_readme(self.current_file, readme_content)
```

Comment:
let's find a different var name that is more descriptive about what it contains.
is it list of dict of the difference outputs where the key is the command name?